quest.yml

```diff
@@ -14,3 +14,5 @@ skills:
 - Operations
 questDependency: docker_localsetup
 slogan: Everything Works If You Bash It Hard Enough
+repository: https://github.com/trywilco/Anythink-Market-Public
```

Comment:
If we add a repository (for shell) we need to add the step that remove the text for the user to run the app with the ports and all